src/components/table/util.js

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+import { css } from 'theme-ui';
+
+const withTruncatedProp = (comp) => {
+  return styled(comp)(
+    ({ theme, color, bg, isTruncated, ...props }) =>
+      css({
+        color,
+        bg,
+        ...props,
+        ...(isTruncated ? {
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+          whiteSpace: 'nowrap',
+          maxWidth: '0px',
+        } : {}),
+      })(theme)
+  );
+};
+
+const getThemeStylesObj = (theme, styleObj, obj) => theme['styles'][styleObj][obj];
+
+export { withTruncatedProp, getThemeStylesObj };
```

Comment:
- [ ] In case if the user wants to see the elipse and expand the content. How can we achieve this?
- [ ] Why don't you think of using our `ReadMore` component?

Reply:
i don't think it is a good way to use Readmore component inside a table